### PR TITLE
fix some errors in "Config Manual.md";

### PR DESCRIPTION
**Pull request by wuzhaojie**
## Description
fix some error in "iotdb\docs\zh\UserGuide\Server\Config Manual.md"

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: docs/zh/UserGuide/Server/Config Manual.md [hunk 1]
```diff
@@ -602,23 +593,6 @@
...
-|改后生效方式|重启服务器生效|
-
-* partition\_interval
-
-|名字| partition\_interval |
-|:---:|:---|
-|描述| 对每个存储组内的数据进行分区的时间范围，单位为秒|
-|取值| LONG |
-|默认值|604800 |
-|改后生效方式|重启服务器生效|
```
Comment (HTHou) [inline docs/zh/UserGuide/Server/Config Manual.md:621]: Why do you remove these? I think these parameters are meaningful....

**Review by HTHou — COMMENTED** [review]
Thanks so much for your contribution! Good job!

Just one question about it.

File: docs/zh/UserGuide/Server/Config Manual.md (hunk 1 above)
Comment (wuzhaojie) [reply docs/zh/UserGuide/Server/Config Manual.md:621]: They are repeated, the above already contains；
image

Comment (wuzhaojie) [reply docs/zh/UserGuide/Server/Config Manual.md:621]: Maybe I was wrong. I just found out that they belong to different modules. I am not familiar with them yet. Please confirm, thank you!
image
image

Comment (HTHou) [reply docs/zh/UserGuide/Server/Config Manual.md:621]: Ok, it makes sense. Thanks.

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**